README.md

```diff
@@ -1 +1,76 @@
-# ensembl-genome-browser
+# Ensembl Genome Browser
+
+This package bundles together compiled version of Ensembl genome browser (GB) along with a javascript communication framework that can be used to embed and control the GB.
+
+
+## Installation
+Using npm:
+```
+npm i --save https://github.com/Ensembl/ensembl-genome-browser
+```
+
+## Creating an instance
+```
+import EnsemblGenomeBrowser, { OutgoingAction, OutgoingActionType } from 'ensembl-genome-browser';
+
+const ensemblGenomeBrowser = new EnsemblGenomeBrowser();
+await ensemblGenomeBrowser.init();
+```
+
+*NOTE:* By default, the genome browser will get embeded to the dom element with id `browser`.
+
+## Methods
+The following are the methods exposed by the GB class.
+
+### send
+This method is used to send instruction messages to the GB in the form of actions (explained below).
+
+Example:
+```javascript
+ensemblGenomeBrowser.send({
+  type: OutgoingActionType.SET_FOCUS;
+  payload: {
+    genomeId: "genome_id";
+  };
+})
+```
+
+
+### subscribe
+This method is used to subscribe to the incoming messages coming from the GB.
+
+The subscribe method takes the following parameters:
+- actionTypes: Array of incoming action types
+- callback`: Function that needs to be called when a particular action is received
```

Comment:
there's an extra backtick after the word "callback"

Reply:
That's done